src/elements/tabs/tab-nav-bar/readme.md

```diff
@@ -0,0 +1,37 @@
+The `sbb-tab-nav-bar` provides a tab-like UI for navigating between routes.
+
+```html
+<sbb-tab-nav-bar>
+  <a href="..." class="sbb-active"> Nav item 1 </a>
+  <a href="..."> Nav item 2 </a>
+  <a href="..."> Nav item 3 </a>
+</sbb-tab-nav-bar>
+```
+
+## States
+
+Add the `sbb-active` class to display the tab as active.
+
+You can also disable a tab by adding the `sbb-disabled` class.
+
+```html
+<sbb-tab-nav-bar>
+  <a href="..." class="sbb-active"> Nav item 1 </a>
+  ...
+  <a class="sbb-disabled" aria-disabled="true"> Nav item 4 </a>
```

Comment:
maybe a short statement that anchors can't be disabled and therefore aria-disabled should be used?